ocp_install_env.sh

```diff
@@ -36,14 +36,39 @@ function extract_installer() {
     extract_command openshift-baremetal-install "$1" "$2"
 }
 
+function save_release_info() {
+    local release_image
+    local outdir
+
+    release_image="$1"
+    outdir="$2"
+
+    oc adm release info --registry-config "$PULL_SECRET_FILE" "$release_image" -o json > ${outdir}/release_info.json
+}
+
+# Gives e.g 4.7.0-0.nightly-2020-10-27-051128
+function openshift_release_version() {
+    jq -r ".config.config.Labels.\"io.openshift.release\"" ${OCP_DIR}/release_info.json
```

Comment:
What about using ".metadata.version" ?

Reply:
AFAICS they return the same value so sure we could do that, any idea why there are both keys?

```
$ jq .metadata.version,.config.config.Labels.\"io.openshift.release\" release_info.json 
"4.7.0-0.nightly-2020-11-10-061115"
"4.7.0-0.nightly-2020-11-10-061115"
```

FWIW I just copied this existing path from the `04` script where it parsed the output of `oc adm release info`

Reply:
AFAIK the `.metadata.version` should describe the current version as per the [Cincinnati upgrade protocol](https://github.com/openshift/cincinnati/blob/master/docs/design/cincinnati.md#update-graph) - but probably as you noted they contain the very same version